server/services/reportService.ts

```diff
@@ -1,14 +1,15 @@
 import R from 'ramda'
 import { IncidentClient } from '../data'
-import type { ReportSummary, IncompleteReportSummary, Report, AnonReportSummary } from '../data/incidentClientTypes'
-import type { LoggedInUser, SystemToken } from '../types/uof'
 
 import logger from '../../log'
 import { PageResponse } from '../utils/page'
 import OffenderService from './offenderService'
 import LocationService from './locationService'
 import ReportLogClient from '../data/reportLogClient'
 import { InTransaction } from '../data/dataAccess/db'
+import type { ReportSummary, IncompleteReportSummary, Report, AnonReportSummary } from '../data/incidentClientTypes'
+
+import type { LoggedInUser, SystemToken } from '../types/uof'
```

Comment:
this presumably can be removed too?

Reply:
No, these were in the file already but at the top